### PR TITLE
facets: fix contribution filter

### PR DESCRIPTION
**Pull request by lauren-d**
* Adds i18n term filter.
* Adapts documents filters in configuration file.
* Adapts tests.

Co-Authored-by: Lauren-D <laurent.dubois@itld-solutions.be>

## Why are you opening this PR?

- Filtering by author does not work in documents search.

## How to test?

- go to public search and try to filter by author.

NB: the behavior must be the same in professional interface 

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

**Review by zannkukai — APPROVED** [review]

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit title proposition: `facets: fix contribution filter`

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Review by rerowep — APPROVED** [review]

File: tests/api/documents/test_documents_rest.py [hunk 1]
```diff
@@ -180,35 +180,35 @@ def test_documents_facets(
     # FILTERS
     # person contribution
     list_url = url_for('invenio_records_rest.doc_list', view='global',
-                       contribution__de='Peter James')
+                       author='Peter James')
```
Comment (jma) [inline tests/api/documents/test_documents_rest.py:183]: Filtering authors with a language parameters seems not be tested.

**Review by jma — APPROVED** [review]

File: tests/api/documents/test_documents_rest.py (hunk 1 above)
Comment (lauren-d) [reply tests/api/documents/test_documents_rest.py:183]: done

**Pull request merged**